user/config_bash.go

```diff
@@ -15,6 +15,7 @@ type BashConfig struct {
 	eConfig
 	Bashpath string `json:"bashpath"` //bash的文件路径
 	Readline string `json:"readline"`
+	ErrNo int
```

Comment:
int or uint8 ?

Reply:
int may good, cause of retval from `command_execute` is int type.